index.html

```diff
@@ -17,5 +17,61 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    
+    <header>
+        
+        <nav>
+           <ul class="list">
+                <li><img src="img/karma-logo.svg" alt="karma logo"/></li>
+                <li><a href="#"><span>Meet Karma</span></a></li>
+                <li><a href="#">How it Works</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <section class="section1">
+            
+            <h1>Introducing Karma</h1>
+            <h2>Bring wifi with you, everywhere you go.</h2>
+            <button class="button">Learn More</button>
+        </section>
+        <section class="section2">
+            <h1 class="icon-title">Everyone needs a little Karma</h1>
+            <div class="divs">
+              <div>
+                 <img src="img/icon-devices.svg" alt="different devices images"/>
+                 <p>Internet for all devices</p>
+              </div>
+              <div>
+                 <img src="img/icon-coffee.svg" alt="hot coffee cup image"/>
+                 <p>Boost your productivity</p>
+              </div>
+              <div>
+                 <img src="img/icon-refill.svg" alt="phone charging image"/>
+                <p>Pay As You Go</p>
+              </div>
+            </div>
+        </section>
+    </main>
+    <hr/>
+    <footer>
+        <div>
+          <h4>Join Us On</h4>
+        </div>
+        <div>
+            <a href="#" class="fa fa-twitter"></a>
+            <a href="#" class="fa fa-facebook"></a>
+            <a href="#" class="fa fa-instagram"></a>
+        </div>
+        <div>
+            <p class="last-para">&copy Karma Mobility, Inc.</p>
+        </div>
+
+    </footer>
 </body>
```

Comment:
Body is a child of the html tag and so should be indented one tab/space. I know this was in the original code you were given, but just thought I would bring it up as its good practise to be aware of this.